### PR TITLE
Fixed a duplicated assignment compared with obj-c code.

### PR DESCRIPTION
**Pull request by canapio**

### CODE REVIEW

**Comment by canapio** [comment]
@jjatie thanks for reply. It’s my mistake. I removed the unnecessary changed file.

File: ChartsDemo/Swift/Demos/LineChart1ViewController.swift [hunk 1]
```diff
@@ -126,7 +126,7 @@ class LineChart1ViewController: DemoBaseViewController {
         set1.valueFont = .systemFont(ofSize: 9)
         set1.formLineDashLengths = [5, 2.5]
         set1.formLineWidth = 1
-        set1.formLineWidth = 15
+        set1.formSize = 15
```
Comment (liuxuan30) [inline ChartsDemo/Swift/Demos/LineChart1ViewController.swift]: why set formSize here but not just delete it?

Comment (jjatie) [reply ChartsDemo/Swift/Demos/LineChart1ViewController.swift]: The objective-c version sets `formSize`

Comment (liuxuan30) [reply ChartsDemo/Swift/Demos/LineChart1ViewController.swift]: oh I don't see it's swift controller.

**Pull request merged**